tag_manager/authorized_api/src/tags.yaml

```diff
@@ -4,32 +4,36 @@ info:
     version: '1.0'
     description: |
         # Tag template options
-        One should send all attributes he wants to be set when using PATCH request.
-        
+        - It is essential to treat the template options as a cohesive object.
+        - When using a PATCH request, remember to include all the attributes you want to be set.
+
         # Deprecations and changes
-        
+
         <details>
         <summary>Deprecated tags</summary>
-        The tags listed below are marked as deprecated and will not be supported soon. 
+        The tags listed below are marked as deprecated and will not be supported soon.
         We strongly discourage using them.
-            
-        | Deprecated tags               |      
+
+        | Deprecated tags               |
         |---                            |
+        | Custom tag (synchronous)      |
+        | A/B Tasty tag                 |
         | Clicktale Tracking Code tag   |
         | DoubleClick Conversion tag    |
-        
+        | Optimizely tag                |
```

Comment:
VWO (Visual Website Optimizer) is missing.

Reply:
Fixed